content/pages/burials-and-memorials/application/530.md

```diff
@@ -10,7 +10,6 @@ layout: page-react.html
       <li><a href="/burials-and-memorials"> Burials and Memorials </a></li>
       <li><a href="/burials-and-memorials/survivor-and-dependent-benefits"> Survivors and Dependents </a></li>
       <li><a href="/burials-and-memorials/survivor-and-dependent-benefits/burial-costs/">Handling Burial Costs (Allowances)</a></li>
-      <li class="active">Apply for Burial Benefits</li>
```

Comment:
Just a question, is there any reason why we can't use the `breadcrumbs.html` partial instead of doing this manually?

If not, then now may be a good time to make the switch since we're touching all those files anyway.